File: 2__C++_Program/3__area_perimeter.cpp
#include <iostream>

using namespace std;

int main() {
    // Declare variables and input length, width
    float length, width;
    cout << "Enter length of a rectangle: "; cin >> length; 
    cout << "Enter width of a rectangle: "; cin >> width;

    // Calculate the area and perimeter
    cout << "Area of a rectangle is: " << length * width << endl;
    cout << "Perimeter of a rectangle is: " << 2*(length + width) << endl;
    return 0;
}

File: 4__Ruby_Program/3__area_perimeter.rb
# Input width, length and print area, perimeter
print "Enter width of a rectangle: "
width = gets.chomp.to_i

print "Enter length of a rectangle: "
length = gets.chomp.to_i

area = width * length
perimeter = 2*(width + length)

print "The area of a rectangle is: #{area}\n"
print "The perimeter of a rectangle is: #{perimeter}"